projects/packages/my-jetpack/_inc/components/product-interstitial/jetpack-ai/product-page.jsx

```diff
@@ -62,7 +64,16 @@ export default function () {
 	const showCurrentUsage = hasPaidTier && ! isFree && usage;
 	const showAllTimeUsage = hasPaidTier || hasUnlimited;
 	const contactHref = getRedirectUrl( 'jetpack-ai-tiers-more-requests-contact' );
-	const newPostURL = 'post-new.php?use_ai_block=1&_wpnonce=' + window?.jetpackAi?.nonce;
+
+	// isRegistered works as a flag to know if the page can link to a post creation or not
+	const newPostURL = isRegistered
```

Comment:
Just a suggestion: since the URL may change between "Create new post" and "Connect to Jetpack [...]", we could call the const a generic `ctaURL`.

Reply:
I wish! I did try to use a ternary, but the optimizations ran at build time will fail. The other option is to use `let` and overwrite, it's just that I haven't seen much use of `let` lately

Reply:
Got it. Actually, my suggestion is just renaming the const from `newPostURL` to `ctaURL` since it may contain other things beyond "new post". But now, after testing the change, I found the exact point where this URL is used, and it's way more "new post" URL than anything else, despite the fact that it can temporarely hold another URL, so I think we can just keep it as is.